receiver/azureeventhubreceiver/azure_formatter_impl.go

```diff
@@ -0,0 +1,204 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package azureeventhubreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/azureeventhubreceiver"
+
+import (
+	"bytes"
+	"encoding/json"
```

Comment:
In order to improve performance, I would suggest using `https://github.com/json-iterator/go` instead.

It can be used as a drop in replacement but note when using it in testing that it doesn't sort keys in a map which can catch people out. (There is a setting that can be turned on for it, or just marshal back into a map)